worlds/sm/__init__.py

```diff
@@ -285,28 +285,29 @@ def APPostPatchRom(self, romPatcher):
                                               "data", "SMBasepatch_prebuilt", "sm-basepatch-symbols.json"))
 
         # gather all player ids and names relevant to this rom, then write player name and player id data tables
-        self.playerIdSet: Set[int] = {0} # 0 is for "Archipelago" server
+        playerIdSet: Set[int] = {0}  # 0 is for "Archipelago" server
         for itemLoc in self.world.get_locations():
+            assert itemLoc.item
```

Comment:
```suggestion
            assert itemLoc.item, f"World of player '{self.world.player_name[itemLoc.player]}' has a loc.item " +
                                 "that is None during generate_output"
```

Reply:
I'm not sure whether that `+` needs a `\` after it, because it's not in parentheses.

Reply:
Yep, it does!